runtime/binding-kafka/src/test/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/CacheMergedIT.java

```diff
@@ -257,6 +257,16 @@ public void shouldFetchMergedMessageValueValid() throws Exception
         k3po.finish();
     }
 
+    @Test
+    @Configuration("cache.options.parameterized.topic.validate.yaml")
+    @Specification({
+        "${app}/merged.fetch.from.parameterized.topic.value.valid/client",
+        "${app}/unmerged.fetch.from.parameterized.topic.value.valid/server"})
+    public void shouldFetchMergedFromParameterizedTopicValid() throws Exception
+    {
+        k3po.finish();
+    }
+
```

Comment:
If we want to make sure the parameterized topic name is selected, then we need to have a test for invalid payload, not valid payload, because this test would still work if we accidentally ignored the topic type model for validation.